build.xml

```diff
@@ -153,16 +153,22 @@
         <equals arg1="${ant.java.version}" arg2="1.8"/>
     </condition>
     <condition property="java.version.11">
-        <not><isset property="java.version.8"/></not>
+        <equals arg1="${ant.java.version}" arg2="11"/>
     </condition>
-    <fail><condition><not><or>
+    <fail message="Unsupported JDK version used: ${ant.java.version}"><condition><not><or>
         <isset property="java.version.8"/>
         <isset property="java.version.11"/>
     </or></not></condition></fail>
 
+    <condition property="arch_x86">
+      <equals arg1="${os.arch}" arg2="x86" />
+    </condition>
+    <condition property="jvm_xss" value="-Xss256k" else="-Xss512k">
+      <isset property="arch_x86" />
+    </condition>
```

Comment:
Do we need to add some comment or something for this condition for completeness or I am tired and this will be obvious and I should leave you alone? :D

Reply:
ill add comment